### DISCUSSION TITLE
Can hooks.client.ts have Handle HandleFetch hooks?

### DISCUSSION THREAD

**Discussion by xmlking** [sveltejs/kit#7518] [Q&A]
I cannot find any reference in the documentation if  `hooks.client.ts` can have `Handle` and `HandleFetch`   handlers. 
when I added, they are not getting called. only `HandleClientError` hook works in `hooks.client.ts`
if that is the case, please clarify in the document. there are may  devs using then on github 
https://github.com/search?q=sveltejs+filename%3A%22hooks.client.ts%22&type=code

If those hooks are not supported in `hooks.client.ts`, what is recommended approach to add `Authorization: Bearer <token>` header for all outgoing fetch calls (to a specific domain) in `+page.ts` or any other client-side fetch code ?

**Comment by teemingc** [comment]
`HandleFetch` only works in `hooks.server`. https://kit.svelte.dev/docs/hooks#server-hooks-handlefetch

You could create a function to help pass the token into the fetch method options
